src/typings/entry.ts

```diff
@@ -16,9 +16,9 @@ export interface ProjectLabelFields {
 export interface AttributeFields {
     id: number;
     data?: {
-        value: {
+        value?: {
             [index: string]: unknown;
-        };
+        } | string;
```

Comment:
This doesn't look right. When is the value object vs string?

Reply:
for conditional widgets attributes, it's an object and rests otherwise